frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -710,6 +710,38 @@ public void testLoadModelFromURL() throws InterruptedException {
         Assert.assertTrue(new File(configManager.getModelStore(), "squeezenet1_1.mar").exists());
     }
 
+    @Test(
+            alwaysRun = true,
+            dependsOnMethods = {"testModelRegisterWithDefaultWorkers"})
+    public void testLoadModelFromFileURI() throws InterruptedException {
+        String curDir = System.getProperty("user.dir");
+        File curDirFile = new File(curDir);
+        String parent = curDirFile.getParent();
+
+        String source = configManager.getModelStore() + "/mnist.mar";
+        String destination = parent + "/modelarchive/mnist1.mar";
+        File sourceFile = new File(source);
+        File destinationFile = new File(destination);
+        String fileUrl = "";
+        try {
+            FileUtils.copyFile(sourceFile, destinationFile);
+            fileUrl = "file://" + parent + "/modelarchive/mnist1.mar";
+        } catch (Exception e) {
+            Assert.assertEquals(e.getClass(), IOException.class);
```

Comment:
Ideally, we should never encounter this exception. If we do, that means something went wrong. It would be better to add the exception with throws.

Reply:
> Please see inline comments.
> 
> The test case for `URL path does not exist` is still missing in both `ModelArchiveTest` and `ModelServerTest`

In ModelArchiveTest.java - We already have a test to check invalid URL - testMalformLocalUrl. 
In ModelServerTest.java - we have added a new test case - testModelWithInvalidFileUri